events/message.js

```diff
@@ -27,7 +28,7 @@ module.exports = async (client, message) => {
 	}
 
 	// Check the message for profanity
-	profanityActions.checkForProfanity(client, message);
+	// TEMP profanityActions.checkForProfanity(client, message);
```

Comment:
Should temps be pushed?

Reply:
I think this is just a personal preference config thing, might not be good to have profanityActions disabled in the base unless it's causing errors with other things.